js/src/components/google-combo-account-card/connected-google-combo-account-card.scss

```diff
@@ -0,0 +1,19 @@
+.gla-google-combo-account-card--connected {
+
+	.gla-account-card__helper {
+		font-style: normal;
+		font-size: $gla-font-base;
+		color: $black;
+
+		em {
+			font-style: italic;
+			color: $gray-700;
+		}
+	}
+
+	.gla-account-card__account_details {
+		display: flex;
```

Comment:
@ankitrox We won't need that if we use `p` instead of `span` in `AccountDetails`